README.md

```diff
@@ -202,28 +202,33 @@ $ pip install -e .[all]
 ```
 This mode installs all the currently publicly available domain extensions - fuseimg as of now, fusedrug will be added soon.
 
-In this mode you can also install and run our end to end examples using:
+To install `FuseMedML` with an included collection of examples install it using:
```

Comment:
I would also include `pip install -e .[all]` to install in interactive mode but without examples